tests/integration/datasets/test_networkrepository_validation.py

```
@@ -0,0 +1,313 @@
+import pytest
+
+from graphforge import GraphForge
+from graphforge.datasets import get_dataset_info, load_dataset
+
+
+@pytest.mark.integration
+@pytest.mark.slow
+class TestNetworkRepositoryValidation:
+    """Validation tests with actual NetworkRepository downloads."""
+
+    def test_karate_dataset_loads_and_validates(self, tmp_path):
```

Ruff (0.14.14)

[warning] Unused method argument: `tmp_path`

(ARG002)